packages/nuxt/src/components/plugins/tree-shake.ts

```
@@ -15,12 +15,192 @@ interface TreeShakeTemplatePluginOptions {
+    if (componentCall.type === 'Identifier') {
+      return componentCall.name
+    } else if (componentCall.type === 'MemberExpression') {
+      return (componentCall.property as Literal).value as string
+    }
+    return (componentCall.arguments[0] as Identifier).name
+  }
+
+  /**
+   * remove a variable declaration within the code
+   */
+  function removeVariableDeclarator (codeAst: Node, name: string, magicString: MagicString, removedNodes: WeakSet<Node>): AcornNode<Node> | void {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)